modules/pam_faillock/main.c

```diff
@@ -106,6 +106,8 @@ args_parse(int argc, char **argv, struct options *opts)
 		}
 	}
 
+	opts->dir = get_tally_dir(opts);
+
```

Comment:
This is not correct. Also I am not sure the #390 is still relevant issue. There were changes in pam_faillock in regards to reading the file recently.